static/elements/chromedash-form-field.js

```diff
@@ -1,14 +1,22 @@
 import {LitElement, css, html} from 'lit';
 import {ALL_FIELDS} from './form-field-specs';
+import {unsafeHTML} from 'lit/directives/unsafe-html.js';
 
 export class ChromedashFormField extends LitElement {
   static get properties() {
     return {
       name: {type: String},
+      value: {type: String},
+      field: {type: String},
+      errors: {type: String},
     };
   }
 
-  static get styles() {
+  static get disabledStyles() {
+    // Since we are using light DOM, for now, these styles are
+    // not used.  See styles in forms.scss instead.
+    // TODO: Check whether we can still use these instead.
```

Comment:
I'm thinking that these styles could be put into chromedash-form-table.js.  But, that could be in a future PR, if it works.

Reply:
I'll try that, but while the docs at https://lit.dev/docs/components/styles/#slotted do say that styles added to the component can affect the component's children, since we are rendering into the light DOM, I'm not sure what happens.